docs/_content/standaard/standaard.md

```diff
@@ -640,6 +729,38 @@ Een `InformatieOject` response van de provider MOET altijd een geldige waarde
 voor `vertrouwelijkheidaanduiding` bevatten. Een client MAG een waarde voor
 `vertrouwelijkheidaanduiding` meesturen.
 
+#### Audit trail
+
+In het DRC is het `EnkelvoudigInformatieObject` het hoofdobject. Alle
+schrijfacties op zaken en gerelateerde objecten MOETEN opgenomen worden in de
+audittrail van het `EnkelvoudigInformatieObject`.
+
+In de audittrail MOETEN de volgende attributen vastgelegd worden:
```

Comment:
Wellicht 1x opschrijven, en dan naar dit kopje verwijzen.  Enige dat afwijkt is de `bron` waarde.
Waarom eigenlijk opnemen hier. De regel

> Zie de API spec voor de overige attributen.

is voldoende toch? Alleen de bron waarde benoemen.

Reply:
Gecondenseerd tot een algemeen stuk dat zegt dat er audittrail logging moet gebeuren op hoofdobjecten van APIs